packages/angular_devkit/build_angular/src/builders/ssr-dev-server/index.ts

```diff
@@ -64,7 +65,7 @@ export function execute(
 ): Observable<SSRDevServerBuilderOutput> {
   let browserSync: typeof import('browser-sync');
   try {
-    browserSync = require('browser-sync');
+    browserSync = createRequire(context.workspaceRoot + '/')('browser-sync');
```

Comment:
`browser-sync` is installed in the project (app), and thus the resolution should happen from the workspaceRoot not from this package.